cluster/examples/kubernetes/ceph/rgw-external.yaml

```diff
@@ -10,9 +10,9 @@ metadata:
 spec:
   ports:
   - name: rgw
-    port: 80
+    port: 8080 # internal port used by the rgw daemon
```

Comment:
Why does this need to be changed to 8080? When using http, 80 should be the default. Why would we make 8080 the default?

Reply:
If this change is still needed, the same change in ceph-object.md is also needed.

Reply:
@travisn : This information about already running object cluster here,  we are not configuring any options. Please refer my below comment for more explanation
Will check ceph-object.md and check any changes is needed or not